src/controllers/commands/CommandController.cpp

```diff
@@ -2936,6 +2937,137 @@ void CommandController::initialize(Settings &, Paths &paths)
+                                "perform that action.";
+            }
+            break;
+
+            case Error::UserNotBroadcaster: {
+                errorMessage +=
+                    "Due to Twitch restrictions, "
+                    "this command can only be used by the broadcaster. "
+                    "To see the list of VIPs you must use the Twitch website.";
+            }
+            break;
+
+            case Error::Unknown: {
+                errorMessage += "An unknown error has occurred.";
+            }
+            break;
+        }
+        return errorMessage;
+    };
+
+    this->registerCommand(
+        "/vips",
+        [formatVIPListError](const QStringList &words,
+                             auto channel) -> QString {
+            switch (getSettings()->helixTimegateVIPs.getValue())
+            {
+                case HelixTimegateOverride::Timegate: {
+                    if (areIRCCommandsStillAvailable())
+                    {
+                        return useIRCCommand(words);
+                    }
+
+                    // fall through to Helix logic
+                }
+                break;
+
+                case HelixTimegateOverride::AlwaysUseIRC: {
+                    return useIRCCommand(words);
+                }
+                break;
+
+                case HelixTimegateOverride::AlwaysUseHelix: {
+                    // do nothing and fall through to Helix logic
+                }
+                break;
+            }
+
+            auto *twitchChannel = dynamic_cast<TwitchChannel *>(channel.get());
+            if (twitchChannel == nullptr)
+            {
+                channel->addMessage(makeSystemMessage(
+                    "The /vips command only works in Twitch channels"));
+                return "";
+            }
+
+            auto currentUser = getApp()->accounts->twitch.getCurrent();
+            if (currentUser->isAnon())
+            {
+                channel->addMessage(makeSystemMessage(
+                    "Due to Twitch restrictions, "  //
+                    "this command can only be used by the broadcaster. "
+                    "To see the list of VIPs you must use the "
+                    "Twitch website."));
+                return "";
+            }
+
+            getHelix()->getChannelVIPs(
+                twitchChannel->roomId(),
+                [channel, twitchChannel](const std::vector<HelixVip> &vipList) {
+                    if (vipList.empty())
+                    {
+                        channel->addMessage(makeSystemMessage(
+                            "This channel does not have any VIPs."));
+                        return;
+                    }
+
+                    auto messagePrefix =
+                        QString("The VIPs of this channel are");
+                    auto entries = QStringList();
+
+                    for (const auto &vip : vipList)
+                    {
+                        entries.append(vip.userName);
```

Comment:
```suggestion
                        entries.append(vip.userLogin);
```
I think user login should be preferred so the clickable name actually yields the correct profile for non-english names

Reply:
Correct, IRC currently returns display_name while webchat does login for `/vips` - but this has been undesired behavior regardless

Reply:
Since the purpose of this PR is purely to port the existing behaviour from the IRC handler to Helix, I'd rather leave this as is. Can be moved to another issue if you believe it's a problem (I know others think it is)

Reply:
Alright. It's worth noting that this IRC change wasn't that long ago (and this fix would close #3524)